admin/c2cgeoportal_admin/views/interfaces.py

```diff
@@ -30,6 +33,11 @@ class InterfacesViews(AbstractViews):
     _model = Interface
     _base_schema = base_schema
 
+    def _base_query(self):
+        return self._request.dbsession.query(Interface). \
+            options(subqueryload('layers')). \
```

Comment:
For me it make no senses to diplay the list of layers in interface (maybe more that 1000 layers in one interface) this may unnecessary increase load on database.

Reply:
If you want to display them, consider limiting your subqueryload to names